CSharpPatterns/Car.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
```

Comment:
Also add a description of what all these classes are responsible for within the implemented patterns